src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -48,20 +49,22 @@
     private final ApplicationProperties props;
     private final InventoryAccountUsageCollector usageCollector;
     private final CloudigradeAccountUsageCollector cloudigradeCollector;
+    private final MetricUsageCollector metricUsageCollector;
     private final UsageSnapshotProducer snapshotProducer;
     private final RetryTemplate retryTemplate;
     private final RetryTemplate cloudigradeRetryTemplate;
 
     private final Set<String> applicableProducts;
 
-    public TallySnapshotController(
+    public TallySnapshotController(//NOSONAR (exceeds allowed 7 params)
```

Comment:
Would prefer `@SuppressWarnings("java:S107")` or whatever appropriate identifier for too many params is.